src/solver/conv_asm_implicit_gemm_gtc_fwd.cpp

```diff
@@ -1534,7 +1534,8 @@ bool ConvAsmImplicitGemmGTCDynamicFwdXdlops::IsApplicable(const ExecutionContext
 
 #if WORKAROUND_SWDEV_306318
     if((problem.kernel_size_h == 1) && (problem.kernel_size_w == 1) && (problem.n_inputs % 8 != 0))
-        return false;
+        if(!miopen::IsEnabled(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_GTC_XDLOPS{}))
+            return false;
```

Comment:
![image](https://user-images.githubusercontent.com/8322173/223009650-7af18374-28bb-424b-b87f-2307911652f8.png)
we have checked this `MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_FWD_GTC_XDLOPS` when entering this function, is there any special reason need to check it again here?